### DISCUSSION TITLE
Script pour obtenir la valeur retournée par un chemin de variable

### DISCUSSION THREAD

**Discussion by StephanBerio** [witsa/synapps#768] [Q&A]
Bonjour, dans un composite je chercher a afficher par exemple une icône pour vrai et une autre pour faux. Je sais le faire avec un additionnel de type booléen, mais je voudrait le faire avec un additionnel de type chemin de variable. Je m'explique si mon chemin de variable est vide je n'affiche rien et  si mon chemin de variable n'est pas vide alors je regarde la valeur sur laquelle il pointe : si la valeur retourne vrai j'affiche une icone si elle retourne faux une autre. Problème : je ne sais pas récupérer dans un script la valeur (true ou false) pointée par mon chemin de variable... un "monCheminDeVariable.value" ne donne rien.
Pouvez vous svp m'aider? Merci d'avance. Amicalement Stéphan

**Comment by WITNicolas** [comment]
Salut Stephan, si tu veux passer par les chemins, il faut alors utiliser un acteur Fournisseur de donnée
En fait seul les acteurs Fournisseur ont le pouvoir d'évaluer un chemin.

Le chemin dans STUDIO n'est qu'une chaine de caractére, l'acteur Fournisseur est justement le moteur d'évaluation.

**Comment by StephanBerio** [comment]
ok merci Nicolas ça marche
